Documentation/bpf/progtypes.rst

```diff
@@ -338,49 +338,88 @@ corresponding network driver name of an interface can be determined as follows:
+|                        +-------------------------+-------------+
+|                        | ixgbe                   | >= 4.12     |
+|                        +-------------------------+-------------+
+|                        | ixgbevf                 | >= 4.17     |
++------------------------+-------------------------+-------------+
+| Marvell                | mvneta                  | >= 5.5      |
+|                        +-------------------------+-------------+
+|                        | mvpp2                   | >= 5.9      |
+|                        +-------------------------+-------------+
+|                        | otx2                    | >= 5.16     |
++------------------------+-------------------------+-------------+
+| Mediatek               | mtk                     | >= 6.0      |
++------------------------+-------------------------+-------------+
+| Mellanox               | mlx4                    | >= 4.8      |
+|                        +-------------------------+-------------+
+|                        | mlx5                    | >= 4.9      |
++------------------------+-------------------------+-------------+
+| Microchip              | lan966x                 | >= 6.1      |
++------------------------+-------------------------+-------------+
+| Microsoft              | hv_netvsc (Hyper-V)     | >= 5.6      |
+|                        +-------------------------+-------------+
+|                        | mana                    | >= 6.0      |
++------------------------+-------------------------+-------------+
+| Netronome              | nfp                     | >= 4.10     |
++------------------------+-------------------------+-------------+
+| Others                 | bonding                 | >= 5.15     |
+|                        +-------------------------+-------------+
+|                        | netdevsim               | >= 4.16     |
+|                        +-------------------------+-------------+
+|                        | tun/tap                 | >= 4.14     |
+|                        +-------------------------+-------------+
+|                        | virtio_net              | >= 4.10     |
+|                        +-------------------------+-------------+
+|                        | xen-netfront            | >= 5.9      |
+|                        +-------------------------+-------------+
+|                        | Veth                    | >= 4.19     |
++------------------------+-------------------------+-------------+
+| QLogic                 | qede                    | >= 4.10     |
++------------------------+-------------------------+-------------+
+| Socionext              | netsec                  | >= 5.3      |
++------------------------+-------------------------+-------------+
+| Solarflare             | SFC Efx                 | >= 5.5      |
++------------------------+-------------------------+-------------+
+| STMicro                | stmmac                  | >= 5.3      |
```

Comment:
Added in v5.13 [see](https://github.com/torvalds/linux/commit/5fabb01207a2d3439a6abe1d08640de9c942945f)